src/main/java/nurseybook/model/NurseyBook.java

```diff
@@ -177,6 +177,14 @@ public void addPossibleGhostTasksWithMatchingDate(LocalDate keyDate) {
         tasks.addPossibleGhostTasksWithMatchingDate(keyDate);
     }
 
+    public void updateTasksOverdueStatus() {
```

Comment:
Would it be possible to write a javadoc here to explain the purpose of the method?

Reply:
Oops, missed that out. Thanks for bringing it to my attention